lib/matplotlib/backends/backend_ps.py

```diff
@@ -344,12 +344,12 @@ def set_font(self, fontname, fontsize, store=True):
                 self.fontname = fontname
                 self.fontsize = fontsize
 
-    def create_hatch(self, hatch):
+    def create_hatch(self, gc, hatch):
```

Comment:
Should probably keep `gc` as the second argument here. Possibly even with fallback to the rcparam if `gc=None` (and thus have `None` as the default)

While it would be a strange thing to call outside of the class, `create_hatch` is not marked as private, and so is technically a public API, so I would err on the side of caution in changing the signature. (i.e. adding an optional parameter is okay, but adding a required parameter or changing order would technically require a deprecation cycle... this is on the edge of being okay to change since it is not likely used many places, but given that it is easy to make it fully back-compatible, inclined to do so)

Reply:
As it isn't usually called outside of the class, why don't we make it private? With the changes you mentioned. 
Or essentially do you want it to work without passing gc to the function?